packages/cxl-ui/src/components/cxl-dashboard-header.js

```diff
@@ -0,0 +1,114 @@
+import { registerGlobalStyles } from '@conversionxl/cxl-lumo-styles/src/utils';
+import cxlDashboardHeaderGlobalStyles from '../styles/global/cxl-dashboard-header-css.js';
+import cxlDashboardHeaderStyles from '../styles/cxl-dashboard-header-css.js';
+
+@customElement('cxl-dashboard-header')
+export class CXLDashboardHeaderElement extends LitElement {
+  static get styles() {
+    return [cxlDashboardHeaderStyles];
+  }
+
+  @property({ type: String }) name = '';
+
+  @property({ type: String }) notificationCount = '6';
+
+  @property({ type: String }) updateTitle = 'What\'s new in CXL';
+
+  @property({ type: String }) lastCourseTitle = 'Advanced experimentation analysis';
+
+  @property({ type: String }) allCourses = 'Browse all courses';
+
+  @property({ type: String }) fastSkillsCourses = 'Browse fast skills';
+
+  @property({ type: String }) createRoadmap = 'Create your personal learning roadmap';
+
+  @property({ type: Boolean }) hasRoadmap = false;
+
+  renderLastCourse() {
+    return this.hasRoadmap
+    ? html`
+      <div class="last-course">
+        <vaadin-button>
+          <div>
+            <span class="subtitle">Continue where you left off</span>
+            <span class="title">${this.lastCourseTitle}</span>
+          </div>
+          <vaadin-icon icon="lumo:arrow-right"></vaadin-icon>
+        </vaadin-button>
+        <div class="progress">
+          <span class="progress-title">Completed 2 lessons of 6 lessons</span>
+          <vaadin-progress-bar value="0.334">Completed 2 lessons of 6 lessons</vaadin-progress-bar>
+        </div>
+      </div>`
+    : nothing;
+  }
+
+  renderStats() {
+    return this.hasRoadmap
+    ? html`
+      <div class="stats">
+        <slot name="stats-desktop"></slot>
+        <vaadin-details theme="cxl-dashboard-header reverse" class="vaadin-details">
+          <div slot="summary">Your roadmap</div>
+          <slot name="stats-mobile"></slot>
+        </vaadin-details>
+      </div>`
+    : nothing;
+  }
+
+  render() {
+    return html`
+      <div class="container">
+        <header>
+          <vaadin-button class="updates">
+            <vaadin-icon class="icon-bell" count="${this.notificationCount}" icon="lumo:bell"></vaadin-icon>
+            ${this.updateTitle}
+            <vaadin-icon icon="lumo:angle-right"></vaadin-icon>
+          </vaadin-button>
+          <div>
+            <span class="subtitle">My dashboard</span>
+            <h1 class="title">Hello ${this.name}!</h1>
+          </div>
+        </header>
+        <section class="content">
+          <div>
+            ${this.renderLastCourse()}
+            <div class="courses">
+              <vaadin-button>
+                ${this.allCourses}
+                <vaadin-icon icon="lumo:arrow-right"></vaadin-icon>
+              </vaadin-button>
+              <vaadin-button>
+                ${this.fastSkillsCourses}
+                <vaadin-icon icon="lumo:arrow-right"></vaadin-icon>
+              </vaadin-button>
```

Comment:
In design these are indeed courses but to make this section more flexible I suggest to rename it to `cta` and buttons to `cta-1`, `cta-2` and possibility to pass URLs too.